### DISCUSSION TITLE
Player flickers

### DISCUSSION THREAD

**Discussion by Dumaiden** [FreeTubeApp/FreeTube#5462] [General]
player flickers, no loading.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Duplicate of https://github.com/FreeTubeApp/FreeTube/issues/5456

**Comment by SJRouge91** [comment]
I ran into the same thing yesterday, even after using an older build, but I solved it by proxying with invidious, maybe it's an issue with local API cause that's what I used beforehand. I hope this helps. **Edit: It looks like someone posted the same solution in an earlier thread and I didn't see it**. Sorry about that.
